docs/get-started/roles-and-permissions/lbac.md

```diff
@@ -22,20 +22,85 @@ PMM supports standard as well as custom labels. PMM automatically assigns standa
 
-1. From the *Main* menu, navigate to <i class="uil uil-cog"></i> *Configuration → Add Service*.
+1. From the **Main** menu, navigate to <i class="uil uil-cog"></i> **Configuration → Add Service**.
 
 2. Select the service you want to add to PMM for monitoring. The page to add the service opens.
 
-3. Enter the details such as *Hostname, Service name, Port, Username, Password,* etc., along with Label or Custom labels.
+    ![!](../../_images/PMM_access_control_add_services.png)
+
+3. Enter the details such as **Hostname, Service name, Port, Username, Password, Max query length**.
+
+    ![!](../../_images/PMM_access_control_add_main_details.png)
+
+4. (Optional) Enter the **Labels** for the service being monitored.
+
+    ![!](../../_images/PMM_access_control_add_labels_to_add.png)
+
+    Starting with PMM 2.40.0, you do not have to explicitly add the cluster label while adding a service to PMM for monitoring. The service name will be automatically assigned to the cluster label.
+
+    ![!](../../_images/PMM_access_control_service_name_assigned_cluster.png)
+
+5. (Optional) Check the **Additional Options** that you want to include, such as:
+
+    - **Skip connection check**: Do not check the connection to the database.
 
- ![!](../../_images/PMM_access_control_add_labels_services.png)
+    - **Use TLS for database connections**: Enabling TLS (Transport Layer Security) for database connections is an important security measure to protect the confidentiality and integrity of data transmitted between the PMM Server and the monitored database instances.
 
+    - **Skip TLS certificate and hostname validation**: For certain purposes, like debugging and testing, it may be necessary to bypass TLS certificate and hostname validation.
+
+    - **Table statistics limit**: Do not collect table statistics if the number of tables in a database exceeds this limit (defaults to 1000).
+
+    - **Disable comments parsing**: Filter out comments in log files or data streams to isolate valid data.
+
+    - **Use performance schema**: Use Performance Schema instead of Slow Query Log (default) for monitoring and diagnosing performance issues in your database.
+ 
  **Using pmm-admin**
 
- You can also assign labels using [pmm-admin](../../details/commands/pmm-admin.md).
\ No newline at end of file
+ You can also assign labels using [pmm-admin](../../details/commands/pmm-admin.md).
+
+## Editing labels
+
+You can edit the labels as follows:
+
+1. From the **Main** menu, navigate to <i class="uil uil-cog"></i> **Configuration → Inventory**.
+
+2. Click on the three dots next to the service you want to edit labels for.
+
+3. Click **Edit**. The **Edit Service** page opens.
+
+4. Edit the labels as per your requirement and click **Save Changes**. The editing service dialogue box opens.
+
+    ![!](../../_images/PMM_access_edit_labels.png)
+
+
+5. Click **Confirm and save changes**. You will be taken back to the **Inventory/Services** page.
+
+### Effect of editing labels
```

Comment:
edited labels doesn't affect existing time-series(metrics) and will be affected only for new time-series(metrics).

Reply:
@BupycHuk So this statement is incorrect:" If labels are used extensively in dashboard configurations, editing labels may require updating existing dashboards to ensure they correctly reference the new labels."

Reply:
it depends on dashboards, if dashboards are based on labels values, then yes, but usually no.